plugins/eos/gs-plugin-eos.c

```diff
@@ -1391,13 +1389,16 @@ app_is_banned_for_personality (GsPlugin *plugin, GsApp *app)
 		return FALSE;
 
 	return ((g_strcmp0 (priv->personality, "es_GT") == 0) &&
-	        g_strv_contains (violent_apps, app_name)) ||
+	        g_strv_contains (restricted_apps, app_name)) ||
```

Comment:
This change to combine `violent_apps` and `adult_apps` means that adult apps are now blocked on `es_GT`. Is that intentional?

Reply:
Yeah, the same sensitivities (won't somebody think of the children!!!!) apply, but when we were actively selling in es_GT, we didn't have Flathub so these adult apps didn't exist. There's no product reason to maintain a separate list, it's just the passage of time. (And lack of ability atm to set a platform-wide filter on extreme violence / extreme nudity / ...)